packages/terra-disclosure-manager/src/DisclosureManager.jsx

```diff
@@ -97,10 +130,24 @@ class DisclosureManager extends React.Component {
   }
 
   openDisclosure(data) {
+    let dimensions = data.dimensions;
+    if (dimensions && !isValidDimensions(dimensions)) {
+      dimensions = defaultDimensions;
```

Comment:
Should we do a console warning  in development mode here if in valid dimensions are passed in, and that it's being defaulted?